closure/goog/math/vec2.js

```diff
@@ -269,6 +269,16 @@ goog.math.Vec2.dot = function(a, b) {
   return a.x * b.x + a.y * b.y;
 };
 
+/**
+ * Returns the detminant of the two vectors.
+ * @param {!goog.math.Vec2} a The first vector.
+ * @param {!goog.math.Vec2} b The second vector.
+ * @return {number} The determinant of the two vectors.
+ */
+goog.math.Vec2.det = function(a, b) {
```

Comment:
The mathematical representation is "det", just like "lerp" is not written "linearInterpolation".